garage/tf/policies/continuous_mlp_policy.py

```diff
@@ -137,23 +139,29 @@ def get_actions(self, observations):
         """Return multiple actions."""
         return self._f_prob_online(observations), dict()
 
-    @property
-    def trainable_vars(self):
-        """Return trainable vars in the network."""
-        return tf.get_collection(
-            tf.GraphKeys.TRAINABLE_VARIABLES, scope=self.name)
+    def get_trainable_vars(self, scope=None):
```

Comment:
This can be moved to a super class.

Reply:
Yes I agree. But I didn't know where to put this. Because I also used this function in continuous_q_fuction.py, so should I put this in garage.tf.Policy or somewhere else?

Reply:
Let's leave dealing with this for later when we get rid of layers.